app/lib/mammoth/channels.rb

```diff
@@ -29,5 +30,27 @@ def find(id)
         JSON.parse(response.body, symbolize_names: true)
       end
     end
+
+    # Subscribe to Channel
+    def subscribe(id, acct)
+      response = HTTP.headers({ Authorization: ACCOUNT_RELAY_AUTH, 'Content-Type': 'application/json' }).post(
+        "https://#{ACCOUNT_RELAY_HOST}/api/v1/channels/#{id}/subscribe?acct=#{acct}"
```

Comment:
The app APIs are calling /api/v3/channels; should I change them to /v1/ ?
Oh; maybe these are the v1 APIs on AcctRelay, so v1 is OK here.

Reply:
still not entirely convinced that _not mirroring_ the endpoints that are proxied was the right call